pages/linux/progress.md

```diff
@@ -0,0 +1,16 @@
+# progress
+
+> Progress Viewer displays/monitors the progress of running coreutils.
+> More information: <https://github.com/Xfennec/progress>.
+
+- Show the progress of running coreutils once:
+
+`progress`
+
+- Continuously monitor the progress of running and upcoming coreutils:
+
+`watch progress -q`
```

Comment:
I am wondering whether this example is helpful. This combines the usage of 2 commands to come up with a new scenario, but adds only the `-q` flag of the progress command. I'd rather explain what the `-q` flag does and keep it at that.

Reply:
Okay, thanks :)